sup3r/pipeline/slicer.py

```diff
@@ -625,14 +652,16 @@ def _get_pad_width(window, max_steps, max_pad, check_boundary=False):
 
         # We add minimum padding to the last slice if the padded window is
         # too small for the generator. This can happen if 2 * spatial_pad +
-        # modulo(grid_size, fwp_chunk_shape) < 4
+        # modulo(grid_size, fwp_chunk_shape) is less than the padding applied
+        # in the first padding layer of the generator
         if (
             check_boundary
             and win_stop == max_steps
-            and (2 * max_pad + win_stop - win_start) < 4
+            and (2 * max_pad + win_stop - win_start) <= min_width
```

Comment:
same comment here, i'd recommend breaking this into semantic variables thats easier for future us to read.